src/pages/_app.tsx

```diff
@@ -17,15 +17,26 @@ const App = ({ Component, pageProps }: AppProps): ReactNode => {
 
             {/* // Import all top-level scripts here: https://nextjs.org/docs/messages/no-script-in-document-page */}
 
-            {/* Triblio "Webpage Personalization" for Sales/Marketing */}
+            {/* Triblio "Webpage Personalization" */}
             <Script
-                data-cfasync="false"
                 type="text/javascript"
                 src="https://tribl.io/h.js?orgId=Yee6bMKj7QSARqAePdE8"
```

Comment:
This appears to be loading in the body by `afterInteractive`, when we anticipate it to load in the head by `beforeInteractive`. Is this expected?

<img width="800" alt="Screen Shot 2022-05-25 at 12 56 26 PM" src="https://user-images.githubusercontent.com/60713139/170331642-42bb150d-dfbc-4d3d-a9d7-bc6869a0431b.png">

Reply:
Hmm, I'm not seeing the same but I did happen to add some ids to the scripts for extra measure even though [Next only requires ids for inline scripts](https://nextjs.org/docs/basic-features/script#:~:text=The%20id%20property%20is%20required%20for%20inline%20scripts%20in%20order%20for%20Next.js%20to%20track%20and%20optimize%20the%20script.). I'm wondering if that maybe fixed it?

---

### Website personalization script (loading in the head)
![Screen Shot 2022-05-25 at 3 05 31 PM](https://user-images.githubusercontent.com/1733936/170349258-ce7c3570-ce3e-4b8c-a3d0-cb02625d8312.png)

---

### Analytics and overlay cards (loading in the body)
![Screen Shot 2022-05-25 at 3 06 40 PM](https://user-images.githubusercontent.com/1733936/170349260-6c7d46a8-e3ef-448e-833a-ed32930f94c0.png)

Reply:
Awesome! I just checked and now they both load in the correct places. Adding the IDs seemed to do the trick.